### PR TITLE
Upgrade readme file

### PR DESCRIPTION
**Pull request by jasiu001**
Related to #35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,37 +10,51 @@ For more information, read the [Helm Broker documentation](https://kyma-project.
 
 To run the project, download these tools:
 
-* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-* [Helm CLI](https://github.com/kubernetes/helm#install)
-* [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) for local installation
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.16
+* [Helm CLI](https://github.com/kubernetes/helm#install) 2.10
```
[the remark this answers is not in the document]
Comment (jasiu001) [reply README.md]: tested. can be :)

File: README.md [hunk 2]
```diff
@@ -10,37 +10,51 @@ For more information, read the [Helm Broker documentation](https://kyma-project.
...
+kind create cluster
 ``` 
 
 2. Install Tiller into your cluster:
 ```bash
-helm init
+helm init --wait
+```
+
+3. Create ServiceAccount for Tiller with admin privileges
```
[the remark this answers is not in the document]
Comment (jasiu001) [reply README.md]: Order is the same: create `serviceaccount`, create `clusterrolebinding`,
the difference is the last command: `helm init` instead `patch deploy`. Do you mean change command? Is there something wrong with `patch`?

File: README.md [hunk 3]
```diff
@@ -10,37 +10,51 @@ For more information, read the [Helm Broker documentation](https://kyma-project.
...
+
+for non-local installation (without kind) use Kubernetes in version 1.15
 
 ## Installation 
 
-To run the Helm Broker, you need a Kubernetes cluster with Tiller and Service Catalog. Follow these steps to set up the Helm Broker on Minikube with all necessary dependencies:
+To run the Helm Broker, you need a Kubernetes cluster with Tiller and Service Catalog. Follow these steps to set up the Helm Broker on Kind with all necessary dependencies:
 
-1. Run the Minikube:
+1. Create local cluster with Kind (making sure the docker is turned on):
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
1. Create a local cluster on Kind:
```

File: README.md [hunk 4]
```diff
@@ -10,37 +10,51 @@ For more information, read the [Helm Broker documentation](https://kyma-project.
...
+
+3. Create ServiceAccount for Tiller with admin privileges
+```bash
+kubectl create serviceaccount --namespace kube-system tiller
+kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+kubectl patch deploy --namespace kube-system tiller-deploy -p '{"spec":{"template":{"spec":{"serviceAccount":"tiller"}}}}'
 ```
 
-3. Install the Service Catalog as a Helm chart:
+4. Install the Service Catalog as a Helm chart:
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
4. Install Service Catalog as a Helm chart:
```

File: README.md [hunk 5]
```diff
@@ -10,37 +10,51 @@ For more information, read the [Helm Broker documentation](https://kyma-project.
...
 ```bash
 helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com
 helm install svc-cat/catalog --name catalog --namespace catalog
 ```
 
-4. Install the Helm Broker chart:
+5. Clone Helm Broker repository
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
5. Clone the Helm Broker repository:
```

File: README.md [hunk 6]
```diff
@@ -116,22 +129,39 @@ Use the following environment variables to configure the Controller component of
...
+The repository has the following structure:
+
+```
+  ├── .github                     # Pull request and issue templates    
+  ├── charts                      # Charts to install by Helm
+  ├── cmd                         # Main applications for project                                     
+  ├── config                      # Configuration file templates or default configs
+  ├── deploy                      # Dockerfiles to build images
+  ├── docs                        # Documentation files
+  ├── hack                        # Various scripts that are used by Helm Broker developers
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
  ├── hack                        # Scripts used by the Helm Broker developers
```

File: README.md [hunk 7]
```diff
@@ -116,22 +129,39 @@ Use the following environment variables to configure the Controller component of
...
+  ├── internal                    # Private application and library code
+  ├── pkg                         # Library code to use by external applications
+  └── test                        # Additional external test apps and test data
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
  └── test                        # Additional external test applications and test data
```

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jasiu001) [reply README.md]: outdated

File: hack/run-dev-kind.sh [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+install::tiller() {
+  shout '- Installing Tiller...'
+  kubectl --namespace kube-system create sa tiller
+  kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+  helm init --service-account tiller --upgrade --wait
+}
+
+tiller::assert_tiller_is_up() {
```
Comment (mszostok) [inline hack/run-dev-kind.sh]: IMO this can be dropped because we already using the `--wait` flag during the init action

File: README.md [hunk 9]
```diff
@@ -10,37 +10,46 @@ For more information, read the [Helm Broker documentation](https://kyma-project.
...
-* [Helm CLI](https://github.com/kubernetes/helm#install)
-* [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) for local installation
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.16
+* [Helm CLI](https://github.com/kubernetes/helm#install) 2.14
+* [Docker](https://docs.docker.com/install/) 19.03 (for local installation)
+* [Kind](https://github.com/kubernetes-sigs/kind#installation-and-usage) 0.5 (for local installation) 
+
+>**NOTE:** For non-local installation, use Kubernetes v1.15.
 
 ## Installation
```
Comment (mszostok) [inline README.md]: We should somewhere describe that you can just execute the
`./hack/run-dev-kind.sh` which will execute all those steps for you. **Now it is a piece of secret knowledge.**

IMO here we can have just info about this script with prerequisite and in the directory where this `./hack/run-dev-kind.sh` have details what is done under the hood and if someone wants to do it manually then can do it.

File: hack/run-dev-kind.sh [hunk 10]
```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+readonly SUPPORTED_KIND_VERSION="v0.5.1"
```
Comment (mszostok) [inline hack/run-dev-kind.sh]: just require this specified in deps_ver.sh

File: hack/run-dev-kind.sh (hunk 8 above)
Comment (jasiu001) [reply hack/run-dev-kind.sh]: I prefer to leave it, if everything will be ok then it is only one iteration and we will be sure tiller is up.

**Pull request merged**